src/main/scala/org/scalameter/persistence/json/scalaMeterModule.scala

```diff
@@ -0,0 +1,112 @@
+package org.scalameter
+package persistence.json
+
+import com.fasterxml.jackson.core.{JsonToken, JsonParser, JsonGenerator}
+import com.fasterxml.jackson.databind._
+import com.fasterxml.jackson.databind.deser.Deserializers
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer
+import com.fasterxml.jackson.databind.jsontype.{TypeSerializer, TypeDeserializer}
+import com.fasterxml.jackson.databind.Module.SetupContext
+import com.fasterxml.jackson.databind.module.SimpleModule
+import com.fasterxml.jackson.databind.ser.Serializers
+import com.fasterxml.jackson.databind.ser.std.StdSerializer
+import com.fasterxml.jackson.databind.`type`.MapLikeType
+import scala.collection.immutable
+import org.scalameter.picklers.Pickler
+
+
+/** Serializer for maps with keys whose are subtype of [[org.scalameter.PicklerBasedKey]].
+ *
+ *  It serializes map values as Base64 encoded byte arrays.
+ *
+ * @param clazz handled type
+ * @tparam MapKey subtype of [[org.scalameter.PicklerBasedKey]]
```

Comment:
Usually we put `tparam` before `param`, to be consistent with the order in the method signature.